prisma/schema.prisma

```diff
@@ -30,5 +31,4 @@ model Post {
   viewCount Int      @default(0)
   author    User?    @relation(fields: [authorId], references: [id])
   authorId  Int?
-  likes     BigInt
```

Comment:
Was there a reason it was removed from the schema?

Reply:
My bad. It was not intentional.